### DISCUSSION TITLE
There are some problems flying the "CF 2.1" (Use LPS - TWR method)

### DISCUSSION THREAD

**Discussion by yuri7579** [bitcraze/discussions#711] [Q&A]
We conducted a task of flying a quadcopter using LPS (Local Positioning System), and although we were able to capture the initial position of the drone accurately, it continues to fly and eventually crashes, as shown in the video.

The video shows the drone hovering and rotating in place, which was the intended operation. However, when we tracked the flight path of the drone, we noticed that while the LPS initially recognizes the coordinates accurately, there are significant errors in the values in the middle of the flight path.

Despite replacing the drone's propellers multiple times, the results were the same. We have considered various potential issues, but have not been able to identify the root cause of the problem. Therefore, we are curious to know what possible problems could be inferred from the video.

https://user-images.githubusercontent.com/129854502/236832597-0d1145c7-3a19-48a1-b660-f52fb0670b16.mp4

**Comment by knmcguire** [comment]
Hi!  I've seen this before. Are you sure that the Crazyflie is pointing towards the positive x-axis of your setup loco system as indicated in the tutorial ? https://www.bitcraze.io/documentation/tutorials/getting-started-with-flying-using-lps/

**Reply by yuri7579** [reply]
It's been resolved thanks to you! Thank you!
